### PR TITLE
Planet-based loading.

### PR DESCRIPTION
**Pull request by monster860**
:cl: monster860
tweak: Z-level loading now occurs during interplanetary travel.
tweak: Interplanetary travel timer bumped up to 1 minute.
/:cl:

### CODE REVIEW

File: code/controllers/subsystem/mapping.dm [hunk 1]
```diff
@@ -83,21 +83,32 @@ var/datum/subsystem/mapping/SSmapping
...
 	..()
 
-/datum/subsystem/mapping/proc/load_star(datum/star_system/star, var/is_initial = 0)
+/datum/subsystem/mapping/proc/clear_navbeacon()
+	var/area/spacearea = locate(/area/space)
+	for(var/datum/sub_turf_block/STB in split_block(locate(1, 1, 1), locate(255, 255, 1)))
+		for(var/turf/T in STB.return_list())
+			for(var/A in T.contents)
+				qdel(A) // Clear everything out, not including docking ports
+			for(var/A in T.contents)
```
Comment (CrAzYPiLoT-SS13) [inline code/controllers/subsystem/mapping.dm]: Wouldn't it be better to check for contents of the turf here, then if something's there, recurse the qdel?

File: code/controllers/subsystem/mapping.dm [hunk 2]
```diff
@@ -116,56 +127,41 @@ var/datum/subsystem/mapping/SSmapping
...
+		var/file = file(map)
+		if(isfile(file))
+			mineral_spawn_override = PL.rings_composition
+			maploader.load_map(file, 1, 1, PL.z_levels[I])
+
+			smooth_zlevel(PL.z_levels[I])
+			world.log << "Z-level [PL.z_levels[I]] for [PL.name] loaded: [map]"
+		else
+			world.log << "Unable to load z-level [PL.z_levels[I]] for [PL.name]! File: [map]"
+		if(PL.spawn_ruins)
```
Comment (CrAzYPiLoT-SS13) [inline code/controllers/subsystem/mapping.dm]: I'm not sure if we should load ruins. They still don't add anything to gameplay value, they just lag the loader.

File: code/controllers/subsystem/mapping.dm [hunk 3]
```diff
@@ -116,56 +127,41 @@ var/datum/subsystem/mapping/SSmapping
...
-	star.navbeacon.main_dock = ftl_start
-	SSstarmap.current_planet = star.navbeacon
-
-	seedRuins(ruins_levels, rand(8,16), /area/space, space_ruins_templates)
+			PL.docks += D
+			PL.name_dock(D, copytext(L.name, 9))
+			if(copytext(L.name, 9) == "main")
+				PL.main_dock = D
+
+	seedRuins(ruins_levels, rand(2,4), /area/space, space_ruins_templates)
```
Comment (CrAzYPiLoT-SS13) [inline code/controllers/subsystem/mapping.dm]: Above comment.

File: code/controllers/subsystem/mapping.dm (hunk 1 above)
Comment (monster860) [inline code/controllers/subsystem/mapping.dm]: Some items are qdel proof, this would never return

Comment (monster860) [inline code/controllers/subsystem/mapping.dm]: Removing ruins is outside the scope of this pr

**Pull request merged**